xcode/Subconscious/Shared/Services/DatabaseService.swift

```diff
@@ -1143,17 +1181,18 @@ final class DatabaseService {
         )
         .compactMap({ row in
             guard
-                let address = row.col(0)?
+                let did = row.col(0)?.toString()?.toDid(),
+                let address = row.col(1)?
                     .toString()?
-                    .toLink()?
```

Comment:
Same question as above

Reply:
Restored it, but answered here: https://github.com/subconsciousnetwork/subconscious/pull/925#discussion_r1353825665